src/app/workspace/cmdinput/textareainput.tsx

```diff
@@ -670,7 +667,7 @@ class TextAreaInput extends React.Component<{ screen: Screen; onHeightChange: ()
                 <textarea
                     key="main"
                     ref={this.mainInputRef}
-                    spellCheck="false"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck .  In this case I think the string "false" is correct.  same for the input box a couple lines below.

Reply:
The type accepted is Booleanish, which accepts bool | "true" | "false". Preact had it as only a bool, so I had to change it. I can revert though